src/versions/validation/workflow.js

```diff
@@ -7,13 +7,19 @@ const contextSchema = Joi.array().unique().items(Joi.string()).single()
 const descriptionSchema = Joi.string().max(2048).allow('', null)
 const booleanExpressionSchema = Joi.string().max(1024).allow('', null)
 
-const orderByFields = [
+const workflowOrderByFields = [
   'name',
   'createdDate',
   'updatedDate',
   'active'
 ]
 
+const workflowLogOrderByFields = [
+  'createdDate',
+  'type',
+  'statusCode',
```

Comment:
I would not enable sorting by `statusCode` for reason mentioned above (missing values).